### ISSUE TITLE
[Polish] [WCAG-EM Report Tool]

### ISSUE THREAD

**Issue by zwiastunsw** [w3c/wai-translations#162]
I would like to  translate into Polish the following resource:
- Page URL: [WCAG-EM Report Tool](https://www.w3.org/WAI/eval/report-tool/)

I have read the information on Translating WAI Documents at https://www.w3.org/WAI/about/translating/

I will wait for confirmation that the resource is ready for translation.

**Comment by remibetin** [comment]
Hi @zwiastunsw 

Thank you very much for volunteering to translate WCAG-EM Report Tool into Polish!

I suppose that it would be your first contribution? If so, please send an email to [group-wai-translations@w3.org](mailto:group-wai-translations@w3.org) expressing your interest in joining the WAI translator volunteers community. I will be happy to guide you through the next steps.

**Comment by zwiastunsw** [comment]
thx, I send e-mail to WAI translator volunteers community.

**Comment by remibetin** [comment]
@zwiastunsw As discussed by email, you can proceed with the translation of the WCAG-EM Report Tool into Polish.

Instructions are available at https://github.com/w3c/wai-wcag-em-report-tool/wiki/How-to-add-a-language

Thanks very much!

**Comment by remibetin** [comment]
Hi @zwiastunsw,

Thanks again for your work!

Is your Pull Request ready for review? Or are you still actively working on it?

**Comment by zwiastunsw** [comment]
Yes, the Pull Request is ready.

Nevertheless, I foresee a problem, as I refer to the WCAG-EM working translation in the links. I declared to translate this document (I have a ready-made version), but the declaration did not meet with any response

**Comment by remibetin** [comment]
Hi @zwiastunsw Thanks very much for the Pull Request! I have added some minor comments in the Pull Request.

The next step is for https://github.com/w3c/wai-wcag-em-report-tool/pull/146 to be reviewed by other volunteers (see [step 5 of the Step-by-Step guide](https://www.w3.org/WAI/about/translating/step-by-step/#review)). I will notify some to know if they would be interested and available to review your translation! If you have volunteers in mind that could help with this, please let me know.

> Nevertheless, I foresee a problem, as I refer to the WCAG-EM working translation in the links.

Indeed, please refer to the original WCAG-EM Working Group Note in the links.

>  I declared to translate this document (I have a ready-made version), but the declaration did not meet with any response

Have you sent an email to [w3c-translators@w3.org](mailto:w3c-translators@w3.org) to declare your intent? (see step 5  at https://www.w3.org/Consortium/Translation/) I cannot find your email [in the archives](https://lists.w3.org/Archives/Public/w3c-translators/).

**Comment by remibetin** [comment]
@zwiastunsw For your information, next step is on my side. I need to check a few things.

I may work on this next week. Feel free to ping me if you do not hear from me by Thursday 14 March.

**Comment by zwiastunsw** [comment]
@remibetin I ping you :)

**Comment by remibetin** [comment]
This translation has been published today 🚀 Thank you for your contribution and patience @zwiastunsw @humanthing-ltd!

See https://www.w3.org/WAI/eval/report-tool/

Your contribution is acknowledged at https://www.w3.org/WAI/eval/report-tool/acknowledgements & https://www.w3.org/WAI/translations/#polski-polish
